api/src/main/java/org/openmrs/Drug.java

```diff
@@ -51,6 +54,7 @@ public class Drug extends BaseChangeableOpenmrsMetadata {
 	@IndexedEmbedded(includeEmbeddedObjectId = true)
 	private Set<DrugReferenceMap> drugReferenceMaps;
 	
+	@NotAudited
```

Comment:
Why annotate this as such?

Reply:
When I enabled auditing for DrugIngredient I could not start the OpenMRS server. I am getting the following error when it is enabled: https://pastebin.com/

Reply:
The pastebin link is empty.

Reply:
sorry, I posted the wrong link. Here is the right one: https://pastebin.com/ySg9jxnm

Reply:
Can we fix this differently? Because we also need to have this field audited. Not so? 😊

Reply:
I found out what was causing the error. This class has a composite key. When using the composite key we need to provide simple objects for id columns.  To resolve this, we can utilize the @MapsId annotation. However,  we have to transition from HBM mapping to JPA annotations.

Should I create a separate PR to switch from HBM mapping to annotations?

Reply:
Yes, create a separate ticket and pull request.

Reply:
@dkayiwa I created a ticket for this: https://openmrs.atlassian.net/browse/TRUNK-6231

Reply:
I have just made it ready for work.